crates/extensions/az_mapper_ext/src/converter.rs

```diff
@@ -48,24 +54,89 @@ impl AzureConverter {
     }
 
     fn try_convert(&mut self, input: &MqttMessage) -> Result<Vec<MqttMessage>, ConversionError> {
-        let default_timestamp = self.add_timestamp.then(|| self.clock.now());
+        if is_bridge_health(&input.topic.name) {
+            Ok(vec![])
+        } else {
+            match self.mqtt_schema.entity_channel_of(&input.topic) {
+                Ok((source, channel)) => self.try_convert_te_topics(source, channel, input),
+                Err(_) => self.try_convert_tedge_health_topics(input),
+            }
+        }
+    }
+
+    fn try_convert_tedge_health_topics(
+        &mut self,
+        input: &MqttMessage,
+    ) -> Result<Vec<MqttMessage>, ConversionError> {
         if is_bridge_health(&input.topic.name) {
```

Comment:
This check is already done inside `try_convert()`. Duplicated.

Reply:
This has to be at the beginning even before we start processing any messages. So, better to keep this inside the `try_convert`